osi_lane.proto

```diff
@@ -678,6 +691,122 @@ message Lane
             //
             optional Identifier successor_lane_id = 2;
         }
+        
+        // A lane's surfaceline (as a list of segments).
+        //
+        // \brief Surfacelines describe the height profile along the lane
+        // Multiple surfacelines can be used to create a certain road shape if needed
+
+        // The surfaceline describes a height profile along the lane on or at a certain offset to the lane boundary.
+        // \image html OSI_LaneBoundaries_And_SurfaceLines.svg "Surfacelines" width=500px
+        //
+        // \note 
+        // sl: surfaceline
+        // lb: lane boundary
+        //
+        // \image html strassenquerschnitt_surfacelines.png "Lane shape surfacelines" width=500px
+        // 
+        // \attention The points of all surfacelines must be set in the 
+        // same ordering (ascending or descending) as the points describing the 
+        // lane boundaries or lane. Example: If the points are deducted from 
+        // a map format, the order of points is recommended to be in line with the 
+        // road coordinate (e.g. s-coordinate in OpenDRIVE). 
+        //
+        // \attention The points describing the surfaceline might be set at
+        // arbitrary distances. This allowes to model certain height changes along the 
+        // lane like speed bumps etc. more detailed while still allow very few points 
+        // at parts of the lane where height does not change much.
+        //
+        // \note The surfaceline is the line that describes the height profile 
+        // along the lane.
+        // 
+        // \attention If more than one surfaceline exists, they are not allowed to cross each other 
+        // or the lane boundaries, otherwise you could have conflicting height information at these points.
+        //
+        // \proposal In order to model curb stones, one surfaceline will be placed at the lower edge
+        // of the curb that belongs to the border lane/driving lane and another one on the upper edge
+        // that will belong to the sidewalk 
+        // (see https://github.com/OpenSimulationInterface/open-simulation-interface/pull/608)
+        //
+        //
+        message SurfaceLine
+        {
+            // The ID of the surfaceline.
+            // Example: sl 3 (see \ref Surfacelines).
+            //
+            // \note Note ID is global unique.
+            //
+            // \rules
+            // is_globally_unique
+            // \endrules
+            //
+            optional Identifier surfaceline_id = 1;
+    
+            // points making up the surfaceline
+            //
+            // If the surfaceline approximates a height curve, the points must be chosen 
+            // in a way that the lateral distance to the ideal line does not exceed 2cm. 
+            // As shown in the following image:
```

Comment:
Would drop the if-part of the sentence because in my understanding a surfaceline always approximates a height curve; also think "lateral distance" is too unspecific:
```suggestion
            // The points must be chosen in a way that the error in vertical direction
            // (for example to an ideal line) does not exceed 2cm, as shown in the following image:
```